test/integration/options/watch.spec.js

```diff
@@ -7,40 +7,40 @@ const helpers = require('../helpers');
 
 describe('--watch', function() {
   describe('when enabled', function() {
-    this.timeout(10 * 1000);
-    this.slow(3000);
+    let tempDir;
```

Comment:
this is just me being fussy.  but most all of our tests use function scope instead of the context to store data.